app/routes/index.ts

```diff
@@ -21,6 +23,12 @@ const handleUnhandledError: Middleware = (
 initializeDb();
 router.use('/user', user);
 router.use('/auth', auth);
+router.use('/pizza', AuthVerify, pizza.createPizza);
+router.use('/get-pizza', AuthVerify, pizza.getPizzas);
+router.use('/get-pizza-by-id/:id', AuthVerify, pizza.getPizzabyId);
```

Comment:
Also consider to not make your routes so messy.
YOu dont need an explanatory route for each method.
A parameter together with an API method is enough